tsconfig.json

```diff
@@ -10,6 +10,9 @@
         "outDir": "./dist",
         "declaration": true,
         "sourceMap": true,
-        "lib": ["es2015", "dom"] // I need DOM because of typedoc..
-    }
+        "lib": ["es2015", "dom", "esnext.asynciterable"]
```

Comment:
I didn't check all files after by previous contribution bringing this name, and that was a mistake: my IDE got lost and couldn't read any config, so it reported a lot of compilation errors. I don't know what's your favorite IDE but I suppose if you browse a lot you also gonna encounter something strange. I experimented with configs inheritance and found out that , in order to get maximum support from IDE (like code completion, jump to `.d.ts`) it's better to have base `tsfonfig.json` which is like “all-in-one”, declaring all libs and covering all files. Then, other configs may inherit from it, reducing libs and sources. So I propose this fix to configs.